submissions/ArthurGorbenko/HTML5-Form/css/styles.css

```diff
@@ -0,0 +1,1223 @@
+* {
+  margin: 0;
+  padding: 0;
+}
+.visually__hidden {
+  position: absolute;
+  overflow: hidden;
+  clip: rect(0 0 0 0);
+  height: 1px;
+  width: 1px;
+  margin: -1px;
+  padding: 0;
+  border: 0;
+}
+body {
+  font-family: "Open Sans", sans-serif;
+  background: #edeef3;
+  color: #707790;
+}
+
+.form {
+  background: #fff;
+  margin: 4em auto;
+  border: 1px solid #cccccc;
+  border-radius: 5px;
+  width: 70%;
+}
+.form-main {
+  margin: 2em auto 2em;
+  display: flex;
+  flex-direction: column;
+  width: 80%;
+}
+.page_header {
+  display: flex;
+  justify-content: space-between;
+}
+.header_logo {
+  height: 150px;
+  width: 150px;
+  margin-top: 5em;
+}
+.page_header_button_back {
+  margin: 10em 0 0 10em;
+  background: #edeef3;
+}
+
+.page_header_button_forward {
+  margin: 10em 10em 0 0;
+  background: #edeef3;
+}
+
+.button {
+  width: 170px;
+  height: 60px;
+  border-radius: 5px;
+  border: 2px solid #cccccc;
+  box-shadow: none;
+  color: #cccccc;
+  font-weight: bold;
+  text-transform: uppercase;
+  transition: all 0.3s;
```

Comment:
Good practice to add an easing function, also try to avoid using `all` in `transition`.

Reply:
Fixed.